ignite/trainer.py

```diff
@@ -50,6 +49,11 @@ def run(self, training_data, max_epochs=1):
         max_epochs: int, optional
             max epochs to train for [default=1]
 
+        Notes
```

Comment:
This exception attribute is now removed right? There's no more need for it, as it is passed to handlers?